### PR TITLE
push

### PR DESCRIPTION
**Pull request by xc112lg**
applying to make ota available to the devices

### CODE REVIEW

File: h870.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+			"oem": "LG",
+			"device": "lucye",
+			"filename": "crDroidAndroid-14.0-20240305-h870-v10.2.zip",
+			"download": "https://github.com/xc112lg/crdroid10.1/releases/download/crDroidAndroid-14.0-20240305/crDroidAndroid-14.0-20240305-h870-v10.2.zip",
+			"timestamp": 1709609412,
+			"md5": "fcff9bd5e976c6a460eb8fc029db2f45",
+			"sha256": "b3bc7cad38db64f1a286a259077df737bab7d28c32871f645730ee72f6f886c9",
+			"size": 1012358508,
+			"version": "10.2",
+			"buildtype": "userdebug",
```
Comment (ttwoxa) [inline h870.json:14]: Must be one of "Testing/Alpha/Beta/Weekly/Monthly"

File: h872.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+			"oem": "LG",
+			"device": "lucye",
+			"filename": "crDroidAndroid-14.0-20240305-h872-v10.2.zip",
+			"download": "https://github.com/xc112lg/crdroid10.1/releases/download/crDroidAndroid-14.0-20240305/crDroidAndroid-14.0-20240305-h872-v10.2.zip",
+			"timestamp": 1709613174,
+			"md5": "59ce5abbd87d793108292ce3d7216e16",
+			"sha256": "67663f28f0fb88acbe07ee71a04cba390cd871081757950cce517c29140b1aed",
+			"size": 1012567338,
+			"version": "10.2",
+			"buildtype": "userdebug",
```
Comment (ttwoxa) [inline h872.json:14]: Must be one of "Testing/Alpha/Beta/Weekly/Monthly"

File: us997.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+			"oem": "LG",
+			"device": "lucye",
+			"filename": "crDroidAndroid-14.0-20240305-us997-v10.2.zip",
+			"download": "https://github.com/xc112lg/crdroid10.1/releases/download/crDroidAndroid-14.0-20240305/crDroidAndroid-14.0-20240305-us997-v10.2.zip",
+			"timestamp": 1709605365,
+			"md5": "2959b51311ff37c34ce974e75cf9294c",
+			"sha256": "95a1af2c0658d85a2ac822379e2a57c9d887a15fe9102ebe735c448282c42641",
+			"size": 1012557628,
+			"version": "10.2",
+			"buildtype": "userdebug",
```
Comment (ttwoxa) [inline us997.json:14]: Must be one of "Testing/Alpha/Beta/Weekly/Monthly"

**Review by ttwoxa — CHANGES_REQUESTED** [review]

File: h870.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "xc112",
+			"oem": "LG",
+			"device": "lucye",
+			"filename": "crDroidAndroid-14.0-20240305-h870-v10.2.zip",
+			"download": "https://github.com/xc112lg/crdroid10.1/releases/download/crDroidAndroid-14.0-20240305/crDroidAndroid-14.0-20240305-h870-v10.2.zip",
```
Comment (ghost) [inline h870.json:8]: Use generated download URL.

File: h870.json (hunk 1 above)
Comment (ghost) [inline h870.json:14]: Must be one of "Testing/Alpha/Beta/Weekly/Monthly"

File: h870.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+			"gapps": "https://sourceforge.net/projects/crdroid/files/other/",
+			"firmware": "",
+			"modem": "",
+			"bootloader": "",
+			"recovery": "https://github.com/xc112lg/crdroid10.1/releases/download/crDroidAndroid-14.0-20240305/recoveryh870.img",
+			"paypal": "https://www.paypal.com/paypalme/xc112xda",
+			"telegram": "",
+			"dt": "https://github.com/LineageOS/android_device_lge_g6-common",
+			"common-dt": "https://github.com/LineageOS/android_device_lge_msm8996-common",
+			"kernel": "https://github.com/LGE-G5-G6-V20/msm8996_lge_kernel/tree/Swan2000-lineage-21"
```
Comment (ghost) [inline h870.json:25]: Use https://github.com/crdroidandroid as repo links.

**Review by ghost — CHANGES_REQUESTED** [review]
Also correct other files.

**Pull request closed without merging**